cmd/node/node.go

```diff
@@ -657,6 +662,43 @@ func (app *SpacemeshApp) startServices() {
 	go app.checkTimeDrifts()
 }
 
+func (app *SpacemeshApp) startAPIServices(postClient api.PostAPI, net api.NetworkAPI) {
+	apiConf := &app.Config.API
+
+	// OLD API SERVICES (deprecated)
+	if apiConf.StartGrpcServer || apiConf.StartJSONServer {
```

Comment:
why keep this code then?

Reply:
Basically, the plan is to gradually enable the new API through this series of PRs, without removing the old API yet. Then once the new API is totally in place, the old one will be removed as part of the last phase. See the plan in #1764. We can't kill off the old API until the new one is fully working. This is what @noamnelke and I had originally discussed.